### PR TITLE
Updating search bar styling

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
Updating search bar styling

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file updates

#### Device Information
IOS simulator, Xperia Z3

#### Screenshots
screen shot 2017-06-14 at 5 21 26 pm
screen shot 2017-06-14 at 5 21 50 pm
screenshot_20170614-185116

### CODE REVIEW

File: app/components/search_bar/search_bar.ios.js [hunk 1]
```diff
@@ -116,9 +116,9 @@ export default class SearchBarIos extends Component {
                     {...this.props}
                     ref='search'
                     placeholderCollapsedMargin={this.state.placeholderWidth}
-                    placeholderExpandedMargin={20}
-                    searchIconCollapsedMargin={this.state.placeholderWidth + 10}
-                    searchIconExpandedMargin={10}
+                    placeholderExpandedMargin={30}
```
Comment (enahum) [inline app/components/search_bar/search_bar.ios.js]: this does not work on iPhone 5

Comment (enahum) [inline app/components/search_bar/search_bar.ios.js]: looks like this

image

Comment (enahum) [inline app/components/search_bar/search_bar.ios.js]: Tried with this values

```js
placeholderCollapsedMargin={this.state.placeholderWidth - 15}
                    placeholderExpandedMargin={30}
                    searchIconCollapsedMargin={this.state.placeholderWidth}
                    searchIconExpandedMargin={15}
```

and that looks like this on an iPhone 5s
image

Comment (enahum) [inline app/components/search_bar/search_bar.ios.js]: my last values on an iPhone 6
image

**Comment by asaadmahmood** [comment]
@enahum pushed with your values.
We can take this for now, and then when you have the time, you can left align the content in the input, and I can look into styling that.

**Pull request merged**